### ISSUE TITLE
Syntax error on Ledger Blue checkGenuine

### ISSUE THREAD

**Issue by hantuzun** [LedgerHQ/blue-loader-python#47]
I plug my Ledger Blue device and checkGenuine` with the following command:

```
$ python -m ledgerblue.checkGenuine --targetId 0x31010004
```

It returns the following error:

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/lib/python2.7/dist-packages/ledgerblue/checkGenuine.py", line 140, in <module>
    secret = getDeployedSecretV2(dongle, bytearray.fromhex(args.rootPrivateKey), args.targetId, args.issuerKey)
  File "/usr/local/lib/python2.7/dist-packages/ledgerblue/checkGenuine.py", line 110, in getDeployedSecretV2
    return ret
UnboundLocalError: local variable 'ret' referenced before assignment
```

That's due to an indentation error introduced in https://github.com/LedgerHQ/blue-loader-python/commit/e8341172299d0073e2e1e77dbaab816d3a3658cd#diff-a38efbe7c6ced1dae1d585bace6a8da0R110

I'll be fixing this with a PR soon.

**Comment by hantuzun** [comment]
Hi Nicolas (@btchip), kindly review this issue.

**Comment by btchip** [comment]
Thanks, will be fixed in 0.1.20
